### PR TITLE
Newbies - Add Section Where everything happens

### PR DESCRIPTION
**Pull request by Drorka**
resolving #110  
@adirkandel @RoyTennenbaum 
- Moved the `UpperTitle.tsx` (created by @develad) from `Maintainers` dir to the `Common` dir, changed it's name to `SectionTitle.tsx`, and updated all the import paths to it.

- Created the where everything happens (`LinksSection.tsx`).
Note that currently the fonts are not displayed on my pc, but they are set in the styles.

desktop
image

image

mobile
image

### CODE REVIEW

**Comment by Drorka** [comment]
@adirkandel @RoyTennenbaum pinging you to check this.
i created here the common section title component, it's in use in the maintainers and newbies page, and can be used in the homepage as well (i saw a new pr addressing this)

File: components/Newbies/LinksSection.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        {linksData.map((linkData, index) => (
+          <div
+            key={index}
+            className="group w-[90vw] min-h-[132px] md:w-full md:max-h-[100px] px-4 py-3 md:px-12 md:py-5 md:mx-20 opacity-80 rounded-lg flex gap-2 justify-between items-center bg-purple-100 hover:bg-purple-200 border border-purple-100 hover:border-purple-700 dark:bg-gray-800 dark:border-gray-800 dark:hover:border dark:hover:border-purple-500 "
+          >
+            <a
+              key={index}
+              href={linkData.link}
+              target="_blank"
+              className="md:group-hover:pointer-events-none"
```
Comment (adirkandel) [inline components/Newbies/LinksSection.tsx:41]: `pointer-events-none` makes this link element not clickable

File: components/Newbies/LinksSection.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+                  <p className="font-normal font-inter text-base">
+                    {linkData.description}
+                  </p>
+                </div>
+              </div>
+            </a>
+
+            <div className="hidden md:block">
+              <a href={linkData.link} target="_blank" rel="noopener noreferrer">
```
Comment (adirkandel) [inline components/Newbies/LinksSection.tsx:66]: I think that it's not a good practice to put 2 links with the same target next to each other.
Let's wrap the whole box with one link element

**Comment by adirkandel** [comment]
@Drorka really good work! left some comments

**Comment by rotembarsela** [comment]
If i may add, the text on the mobile screen suppost to be on the left side instead of the right side above the line, you can check my lastest PR and see how I writed that, the bullet on FAQ page is a little square

**Comment by Drorka** [comment]
@adirkandel 
Thanks for the feedback!

I'm addressing both your comments here because they're related.
I figured that the desired behavior of each element is:
- on desktop view - only the "play" button on the left is clickable (which is not visible on mobile)
- on mobile view - the entire element is clickable 
In order to achieve this, I created 2 links with the same target, but only one is active in each view (desktop / mobile).
That's why I used the `pointer-events-none` - to disable the link of the entire element when in desktop view.

Hope it's clearer now, let me know if you think it should behave differently .

**Comment by adirkandel** [comment]
@Drorka as i see it there is no benefit in clicking on a specific element on desktop and on mobile it acts differently.
One of the ux rules is consistency so i think the entire element should be clickable on both devices to keep it consistent and easy to get to the link.

**Comment by Drorka** [comment]
> @Drorka as i see it there is no benefit in clicking on a specific element on desktop and on mobile it acts differently. One of the ux rules is consistency so i think the entire element should be clickable on both devices to keep it consistent and easy to get to the link.

@adirkandel sure thing, i'll update it so the entire element will be clickable both on desktop and mobile, and get rid of the second link.

**Comment by Drorka** [comment]
@adirkandel all done

**Comment by Drorka** [comment]
@adirkandel resolved them (:

**Review by RoyTennenbaum — APPROVED** [review]
Perfect! thanks for fixing the small things Adir mentioned and resolving conflicts.

Merging :)

**Pull request merged**